crates/stackable-operator/src/utils/kubelet.rs

```diff
@@ -0,0 +1,69 @@
+use http;
+use k8s_openapi::api::core::v1::Node;
+use kube::{
+    Api,
+    api::{ListParams, ResourceExt},
+    client::Client,
+};
+use serde::Deserialize;
+use snafu::{OptionExt, ResultExt, Snafu};
+
+use crate::commons::networking::DomainName;
+
+#[derive(Debug, Snafu)]
+pub enum Error {
+    #[snafu(display("failed to list nodes"))]
+    ListNodes { source: kube::Error },
+    #[snafu(display("failed to build proxy/configz request"))]
```

Comment:
I first though this is an "either proxy or configz", which confused me. This makes it a bit more clear that this is a HTTP path.
```suggestion

    #[snafu(display("failed to build \"/proxy/configz\" request"))]
```